xml2kinto/tests/test_id_generator.py

```diff
@@ -11,3 +11,23 @@ def test_create_id():
             "children": ["foo", "bar", "foobar"]
         }
     }) == 'ef636b94-97a2-3c5e-5a64-22e23cf28e8f'
+
+
+def test_create_id_ignores_enabled_flag():
+    """It should handle strings, list, int and dict."""
```

Comment:
This docstring is a bit misleading, does it mean it should "use" everything but booleans? Why?